src/shared/components/case-editor/services/work-allocation.service.ts

```diff
@@ -1,11 +1,12 @@
 import { Injectable } from '@angular/core';
-import { Observable, of, throwError } from 'rxjs';
+import { Observable, throwError } from 'rxjs';
 import { catchError, map } from 'rxjs/operators';
 
 import { AbstractAppConfig } from '../../../../app.config';
-import { TaskSearchParameter, TaskSearchParameters } from '../../../domain';
+import { TaskSearchParameter } from '../../../domain';
 import { UserDetails } from '../../../domain/user/user-details.model';
```

Comment:
Very minor comment - Nitpicking here but looks like this ordering will cause a lint warning

Reply:
Thank you @connorpgpmcelroy, I have now applied the fix.